### DISCUSSION TITLE
Connect multiple HX711 and relay measurements over MQTT

### DISCUSSION THREAD

**Discussion by thomas-bingel** [arendst/Tasmota#15881] [Ideas]
Hello everyone, 

I tried to connect two HX711 to the D1 but when configured only one shows up in the overview menu. 

This feature request (https://github.com/arendst/Tasmota/issues/7784) was closed, but it seems there are more people interested in this feature. 

Are there any updates on how multiple HX711 can be connected to the D1?

Lots of thanks to the community!
  Thomas

**Comment by sfromis** [comment]
As I already pointed out there, the way to connect two HX711 is to have two D1 boards. Cheap and easy solution.

Someone with the interest and time could sit down to do the work to extend the existing support to more than one HX711. There is no sort of planned schedule where you can get updates.

**Comment by thomas-bingel** [comment]
Thank you for your reply. Do you know the code and have an estimate on how long the coding would take? If it is not to much work, maybe I can take a look at it.

**Reply by sfromis** [reply]
I know where the code is: https://github.com/arendst/Tasmota/blob/development/tasmota/tasmota_xsns_sensor/xsns_34_hx711.ino

And a few other drivers do support more than one sensor of the same type, but that's mostly for simpler cases.
